homeassistant/components/lifx/__init__.py

```diff
@@ -57,7 +57,7 @@
 )
 
 
-PLATFORMS = [Platform.BINARY_SENSOR, Platform.BUTTON, Platform.LIGHT]
+PLATFORMS = [Platform.BINARY_SENSOR, Platform.BUTTON, Platform.SELECT, Platform.LIGHT]
```

Comment:
```suggestion
PLATFORMS = [Platform.BINARY_SENSOR, Platform.BUTTON, Platform.LIGHT, Platform.SELECT]
```

Reply:
If `Platform.LIGHT` isn't last, Home Assistant uses the wrong name for the device.

Reply:
That seems like a bug in the integration. Its not expected to matter which order that are loaded in.  We usually load them in ABC order for convention.

Reply:
It looks like it is using `.label`. 

Is that not populated until `light.py` has been loaded?

Reply:
Let me test and adjust accordingly. It may have been quietly resolved since the last time I changed the platform order. The label should be known given how discovery occurs.